tests/test_metadata.py

```diff
@@ -1,7 +1,14 @@
-import ansys.tools.report as report
+import ansys.tools as ansys_tools
+import ansys.tools.report as ansys_tools_report
 
 
-def test_pkg_version():
+def test_pkg_version_method():
     """Test the correct functioning of the version() method."""
```

Comment:
@RobPasMue Do we still need this `version()` method as we now have the `__version__` attribute?

Reply:
I mean, it's not "necessary" per se... But I'd like to give the option to users to choose their preferred option.